challenge.js

```diff
@@ -42,7 +42,7 @@
  */
 
 // ANSWER
-
+document.querySelector("E");
```

Comment:
Partial credit for proper usage of `document.querySelector`. This question was more interested in the CSS selector string, though. Here's some help with CSS selector strings:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

1/4